src/app/inventory/d2-stores.ts

```diff
@@ -141,12 +150,111 @@ export function loadStores(): ThunkResult<DimStore[] | undefined> {
     }
 
     $featureFlags.clarityDescriptions && dispatch(loadClarity()); // no need to await
+    await dispatch(loadNewItems(account));
     const stores = await dispatch(loadStoresData(account));
     return stores;
   };
 }
 
-let latestDateLastPlayedTimestamp = 0;
+const BUNGIE_CACHE_TTL = 15_000;
+
+let minimumCacheAge = Number.MAX_SAFE_INTEGER;
```

Comment:
so is this like, minimum observed time between updates? just as trivia rn?

Reply:
Yeah I could remove it, it was just so if I cranked down the refresh rate what would I actually get.